tests/cephfs/cephfs_utils.py

```diff
@@ -65,6 +65,89 @@ def _setup_crefi(node):
         # Setup Crefi pre-requisites : pyxattr
         node.exec_command(sudo=True, cmd="pip3 install pyxattr", long_running=True)
 
+    def prepare_clients(self, clients, build):
+        for client in clients:
+            node = client.node
+            if node.pkg_type == "rpm":
+                out, rc = node.exec_command(
+                    sudo=True, cmd="rpm -qa | grep -w 'attr\\|xattr'"
+                )
+                output = out.read().decode()
+                output.split()
+                if "attr" not in output:
+                    node.exec_command(sudo=True, cmd="yum install -y attr")
+                node.exec_command(
+                    sudo=True, cmd="yum install -y gcc python3-devel", check_ec=False
+                )
+
+                if "xattr" not in output:
+                    pkgs = [
+                        "@development",
+                        "rh-python36",
+                        "rh-python36-numpy rh-python36-scipy",
+                        "rh-python36-python-tools rh-python36-python-six",
+                        "libffi libffi-devel",
+                    ]
+                    if build.endswith("7") or build.startswith("3"):
```

Comment:
I am not sure about this logic... I understand `startswith(3)` but not sure why are we checking `endswith("7")`